app/src/ui/lib/highlight-text.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react'
+
+// broken for SFCs
```

Comment:
Is there a tracking issue for this somewhere? If so, we should add that in as a comment for the next person looking at this code.

Reply:
Yep: https://github.com/Microsoft/tslint-microsoft-contrib/issues/339